src/main/app/access-code/AccessCodePostController.ts

```diff
@@ -0,0 +1,54 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { getNextStepUrl } from '../../steps';
+import { getCaseApi } from '../case/CaseApi';
+import { CITIZEN_LINK_APPLICANT_2 } from '../case/definition';
+import { AppRequest } from '../controller/AppRequest';
+import { AnyObject, PostController } from '../controller/PostController';
+import { Form } from '../form/Form';
+
+@autobind
+export class AccessCodePostController extends PostController<AnyObject> {
```

Comment:
bonus points for putting this file in the correct folder (and not the controller folder)